test/helpers.ts

```diff
@@ -183,3 +184,15 @@ test('shouldFeatureRun', t => {
 		exclude: yesNo,
 	}), 'If any `exclude` is true, then it should not run, regardless of `asLongAs` and `include`');
 });
+
+test('getItemsBetween', t => {
```

Comment:
Thanks for writing some tests! Could you move them to a separate `get-items-between.test.ts` file and place it next to the helper? I think that's what causing the conflict